### ISSUE TITLE
Provide an option to use `envoyproxy/envoy-contrib` images

### ISSUE THREAD

**Issue by zd9KgA** [tetratelabs/archive-envoy#37]
I'd like to be able to use `func-e` in connection with Envoy configuration that required contributed functionalities 
(such as experimenting with PostgresProxy).

An option (e.g. via command line switch) to toggle between the regular behavior and using the extended image would be helpful (to me).

My current alternative is not to use `func-e`, but to work explicitly with the images.

**Comment by zd9KgA** [comment]
@codefromthecrypt Thanks for the hint. Until a corresponding archive is maintained, I'll try to craft and provide a corresponding JSON file locally.

**Comment by codefromthecrypt** [comment]
yep this was an intentional design of ours. by using a json URL people can reduce the choices or expand them per site concerns or when there's a gap like this!

note: this is the schema if it helps https://archive.tetratelabs.io/release-versions-schema.json
